tests/tests_certificate.yml

```diff
@@ -0,0 +1,133 @@
+---
+- name: Test PostgreSQL server with ssl support using certificate role
+  hosts: all
+
+  tasks:
+    - name: Test PostgreSQL server with certificate in non default path
+      block:
+        - name: Deploy postgresql
+          include_role:
+            name: linux-system-roles.postgresql
+          vars:
+            postgresql_certificates:
+              - name: /tmp/tmp_cert
```

Comment:
The problem is selinux:
```
type=AVC msg=audit(1670516051.370:1173): avc:  denied  { open } for  pid=18100 comm="postmaster" path="/tmp/tmp_cert.crt" dev="vda4" ino=26413864 scontext=system_u:system_r:postgresql_t:s0 tcontext=system_u:object_r:certmonger_tmp_t:s0 tclass=file permissive=0
```
The certificate system role (via certmonger) by default manages certs in the defined system directory `/etc/pki/tls/certs` and private key files in `/etc/pki/tls/private`.  There is SELinux policy set up for these directories.  If you want to use another directory like `/tmp` with the certificate role, you'll need to label those directories for SELinux.

Does the postgresql role need to support the use of the certificate system role with non-default directories?

Reply:
> Ok, it could be properly documented in the readme.

https://github.com/linux-system-roles/certificate/#certmonger-and-selinux

> To use only the default path. So I will add the check whether the name doesn't contain an absolute path.

You can use paths other than the default path.  But if you do, you must use the selinux role to properly label those paths.

My question is - Does the postgresql role need to be able to use the certificate role to issue certificates to non-default paths?

Reply:
It isn't necessary to support this case.

Reply:
ok - then I guess remove this test